executor/executor_test.go

```diff
@@ -3441,32 +3441,6 @@ type testSuiteWithCliBaseCharset struct {
 	testSuiteWithCliBase
```

Comment:
forgot this testSuite

Reply:
we need this testSuite, we can not use `testSuiteWithCliBase` directly, just like the other tests